src/components/Avatar/Avatar.scss

```diff
@@ -0,0 +1,17 @@
+@import "~styles/main";
+
+.dotted {
```

Comment:
Haven't I shown you how to do this in call? You can also checkout the stackblitz [link](https://stackblitz.com/edit/react-cnpjqt).

Reply:
Done